src/pages/tables/admins-table.html

```diff
@@ -0,0 +1,778 @@
+                        </tr>
+                        <tr>
+                          <td class="py-1">
+                            <img src="../../assets/images/faces/face5.jpg" alt="image" />
+                          </td>
+                          <td>
+                            Edward
+                          </td>
+                          <td>
+                            <div class="progress">
+                              <div class="progress-bar bg-danger" role="progressbar" style="width: 35%"
+                                aria-valuenow="35" aria-valuemin="0" aria-valuemax="100"></div>
+                            </div>
+                          </td>
+                          <td>
+                            $ 160.25
+                          </td>
+                          <td>
+                            May 03, 2015
+                          </td>
+                        </tr>
+                        <tr>
+                          <td class="py-1">
+                            <img src="../../assets/images/faces/face6.jpg" alt="image" />
+                          </td>
+                          <td>
+                            John Doe
+                          </td>
+                          <td>
+                            <div class="progress">
+                              <div class="progress-bar bg-info" role="progressbar" style="width: 65%" aria-valuenow="65"
+                                aria-valuemin="0" aria-valuemax="100"></div>
+                            </div>
+                          </td>
+                          <td>
+                            $ 123.21
+                          </td>
+                          <td>
+                            April 05, 2015
+                          </td>
+                        </tr>
+                        <tr>
+                          <td class="py-1">
+                            <img src="../../assets/images/faces/face7.jpg" alt="image" />
+                          </td>
+                          <td>
+                            Henry Tom
+                          </td>
+                          <td>
+                            <div class="progress">
+                              <div class="progress-bar bg-warning" role="progressbar" style="width: 20%"
+                                aria-valuenow="20" aria-valuemin="0" aria-valuemax="100"></div>
+                            </div>
+                          </td>
+                          <td>
+                            $ 150.00
+                          </td>
+                          <td>
+                            June 16, 2015
+                          </td>
+                        </tr>
+                      </tbody>
+                    </table>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>-->
+        <div class="main-panel">
+          <div class="content-wrapper">
```

Comment:
add the following code snippet directly after <div class="content-wrapper">
```html
<div class="row py-3 justify-content-between align-items-center">
              <h2 class="fw-bold col-lg-auto">Admins</h2>
              <div class="col-lg-auto">
                <!-- Search Bar start -->
                <div class="search-bar">
                  <form action="">
                    <div class="row g-1 justify-content-lg-end justify-content-start">
                      <div class="col-6 col-lg-3 form-floating">
                        <input type="text" class="form-control" id="title">
                        <label for="title">Username</label>
                      </div>
                      <div class="col-6 col-lg-3 form-floating">
                        <input type="text" class="form-control" id="title">
                        <label for="title">Email</label>
                      </div>
                      <button class="col-auto btn border-0 btn-md" type="submit">
                        <i class="icon-search fs-5"></i>
                      </button>
                    </div>
                  </form>
                </div>
                <!-- Search Bar ends -->
              </div>
            </div>
```

Reply:
ok